openfisca_core/parameters.py

```diff
@@ -79,40 +80,52 @@ def __init__(self, message, file = None, traceback = None):
         super(ParameterParsingError, self).__init__(message)
 
 
-class AbstractParameter(object):
-    allowed_keys = None
-    _data_type = None
+class ValidableParameter(object):
```

Comment:
1. Is there a better name for the class ? ProperlyFormatedParameter ?
2. Is there a need for this extra class ? It seem to only do security checks, and it seem they could be contained in the Datable parameter class. Can there be a parameter that is not datable ?

Reply:
Removed the two common classes